lib/traject/regex_split.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# work-around for https://github.com/jruby/jruby/issues/4868
```

Comment:
@cdmo good call to move these to its own file 👍

Reply:
yeah I kind of had to do it because including `psulib_config.rb` into the marc processing class was a no go.